### PR TITLE
Redo fixture information display using Rich

### PR DESCRIPTION
**Pull request by JoshKarpel**
@darrenburns you're not the only one who's been playing with Rich....

Here's what it looks like right now. I mostly tried to mimic how it was before, but Rich provides so many options that there's surely room for improvement.

image

image

(I ran `make prep` and got a bunch of diffs from `black` - maybe it got missed on a previous PR?)

### CODE REVIEW

**Comment by darrenburns** [comment]
This looks great Josh, thanks! I'll try and give it a look over this week (most likely Friday) 🙂

File: ward/terminal.py [hunk 1]
```diff
@@ -387,8 +399,8 @@ def output_all_test_results(
             console.print()
         return all_results
 
-    def print_divider(self):
-        console.print(Rule(style="muted"))
+    def print_divider(self) -> None:
+        print_divider()
```
Comment (darrenburns) [inline ward/terminal.py:403]: Any reason in particular for moving this out? The function only seems to be used here.

**Review by darrenburns — APPROVED** [review]
One minor non-blocking comment, but looks great. Thanks for taking the time to do this, it's a big improvement in readability.

**Comment by JoshKarpel** [comment]
I think I was originally going to use `print_divider` [here](https://github.com/JoshKarpel/ward/blob/46b808cb4609316011bb37926586e7841658963f/ward/terminal.py#L669), but ended up going a different route. Reverted!

**Pull request merged**